build/patches/Add-an-always-incognito-mode.patch

```diff
@@ -760,6 +764,108 @@ diff --git a/chrome/common/pref_names.h b/chrome/common/pref_names.h
+                bool off_the_record,
++               bool always_incognito_enabled,
+                bool store_last_modified,
+                bool restore_session);
+ 
+@@ -86,6 +87,7 @@ class PrefProvider : public UserModifiableProvider {
+   raw_ptr<PrefService> prefs_;
+ 
+   const bool off_the_record_;
++  const bool always_incognito_enabled_;
+ 
+   bool store_last_modified_;
+ 
+diff --git a/components/content_settings/core/browser/host_content_settings_map.cc b/components/content_settings/core/browser/host_content_settings_map.cc
+--- a/components/content_settings/core/browser/host_content_settings_map.cc
++++ b/components/content_settings/core/browser/host_content_settings_map.cc
+@@ -238,6 +238,7 @@ const char* ContentSettingToString(ContentSetting setting) {
+ HostContentSettingsMap::HostContentSettingsMap(
+     PrefService* prefs,
+     bool is_off_the_record,
++    bool always_incognito_enabled,
+     bool store_last_modified,
+     bool restore_session)
+     : RefcountedKeyedService(base::ThreadTaskRunnerHandle::Get()),
+@@ -246,6 +247,7 @@ HostContentSettingsMap::HostContentSettingsMap(
+ #endif
+       prefs_(prefs),
+       is_off_the_record_(is_off_the_record),
++      always_incognito_enabled_(always_incognito_enabled),
+       store_last_modified_(store_last_modified),
+       allow_invalid_secondary_pattern_for_testing_(false) {
+   TRACE_EVENT0("startup", "HostContentSettingsMap::HostContentSettingsMap");
+@@ -257,7 +259,7 @@ HostContentSettingsMap::HostContentSettingsMap(
+   policy_provider->AddObserver(this);
+ 
+   auto pref_provider_ptr = std::make_unique<content_settings::PrefProvider>(
+-      prefs_, is_off_the_record_, store_last_modified_, restore_session);
++      prefs_, is_off_the_record_, always_incognito_enabled_, store_last_modified_, restore_session);
+   pref_provider_ = pref_provider_ptr.get();
+   content_settings_providers_[PREF_PROVIDER] = std::move(pref_provider_ptr);
+   user_modifiable_providers_.push_back(pref_provider_);
+diff --git a/components/content_settings/core/browser/host_content_settings_map.h b/components/content_settings/core/browser/host_content_settings_map.h
+--- a/components/content_settings/core/browser/host_content_settings_map.h
++++ b/components/content_settings/core/browser/host_content_settings_map.h
+@@ -80,6 +80,7 @@ class HostContentSettingsMap : public content_settings::Observer,
+   // profile or a guest session.
+   HostContentSettingsMap(PrefService* prefs,
+                          bool is_off_the_record,
++                         bool always_incognito_enabled,
+                          bool store_last_modified,
+                          bool restore_session);
+ 
+@@ -309,6 +310,7 @@ class HostContentSettingsMap : public content_settings::Observer,
+ 
+   // Whether this settings map is for an incognito or guest session.
+   bool IsOffTheRecord() const { return is_off_the_record_; }
++  bool IsAlwaysIncognito() const { return always_incognito_enabled_; }
```

Comment:
Unused?

Reply:
ops... yes.
I used it to check with log(), then I forgot to take it off.

Reply:
Ok, I will remove it.